src/main/java/org/zalando/nakadi/service/AuthorizationValidator.java

```diff
@@ -86,18 +84,11 @@ private void checkAuthAttributesNoDuplicates(final Map<String, List<Authorizatio
         }
     }
 
-    private void checkAuthAttributesAreValid(final Map<String, List<AuthorizationAttribute>> allAttributes)
+    private void checkAuthAttributesAreValid(final Resource resource)
             throws UnableProcessException, ServiceTemporarilyUnavailableException {
         try {
-            final String errorMessage = allAttributes.values().stream()
-                    .flatMap(Collection::stream)
-                    .filter(attr -> !authorizationService.isAuthorizationAttributeValid(attr))
-                    .map(attr -> String.format("authorization attribute %s:%s is invalid",
-                            attr.getDataType(), attr.getValue()))
-                    .collect(Collectors.joining(", "));
-
-            if (!Strings.isNullOrEmpty(errorMessage)) {
-                throw new UnableProcessException(errorMessage);
+            if (authorizationService.isAuthorizationForResourceValid(resource)) {
+                throw new UnableProcessException("Authorization Attributes are not valid");
```

Comment:
I think this is a change that makes user experience worse. 

Before this change user was informed which attributes are invalid. Now the user will only be informed about a fact that they are invalid.

I think method `isAuthorizationForResourceValid` can't return just a boolean as `isAuthorizationAttributeValid` does. I think it can return a list of errors or just throw an exception that will contain information about specific attributes that are invalid.

Reply:
added in the commit